com.ibm.streamsx.topology/opt/python/include/splpy_setup.h

```diff
@@ -131,11 +140,23 @@ class SplpySetup {
           SplpyGeneral::loadFunction("streamsx.spl.types", "_get_timestamp_tuple"));
        SplpyGeneral::decimalClass(
           SplpyGeneral::loadFunction("decimal", "Decimal"));
+   }
+
+   static void setupSplNull() {
+       SplpyGIL lock;
 
-       PyObject *fn = SplpyGeneral::loadFunction("streamsx.spl.types", "null");
-       SplpyGeneral::isSplNull(
-          SplpyGeneral::pyCallObject(fn, NULL));
-       Py_DECREF(fn);
+       // Get a pointer to SPL null.
+       PyObject *null = SplpyGeneral::callFunction("streamsx.spl.types", "null", NULL, NULL);
```

Comment:
Maybe for readability use variable name of `splnull` or `spl_null` to avoid mentally thinking that this is `NULL` or `null` from Java.